packages/messaging/ios/RNFBMessaging/RNFBMessaging+UNNotificationServiceExtension.m

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2016-present Invertase Limited & Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this library except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#import "RNFBMessaging+UNNotificationServiceExtension.h"
+#import "FirebaseMessaging.h"
+
+API_AVAILABLE(ios(10.0))
```

Comment:
Interesting - I had thought RNFB moved to iOS10 as a minimum but it still supports 9 right now: https://github.com/invertase/react-native-firebase/blob/master/packages/messaging/RNFBMessaging.podspec#L28 - it's Notifee and React-Native itself (with version 0.63) that have moved to 10 already